service/src/main/kotlin/fi/espoo/evaka/calendarevent/CalendarEventController.kt

```diff
@@ -535,6 +542,73 @@ class CalendarEventController(
         }
     }
 
+    @PostMapping("/employee/calendar-event/clear-survey-reservations-for-child")
+    fun clearEventTimesInEventForChild(
+        db: Database,
+        user: AuthenticatedUser.Employee,
+        clock: EvakaClock,
+        @RequestBody body: CalendarEventTimeClearingForm,
+    ) {
+        val removedIds =
+            db.connect { dbc ->
+                dbc.transaction { tx ->
+                    accessControl.requirePermissionFor(
+                        tx,
+                        user,
+                        clock,
+                        Action.CalendarEvent.UPDATE,
+                        body.calendarEventId,
+                    )
+                    val eventTimesToRemove =
+                        tx.getCalendarEventTimesByChildAndEvent(body.childId, body.calendarEventId)
+                    accessControl.requirePermissionFor(
+                        tx,
+                        user,
+                        clock,
+                        Action.CalendarEventTime.UPDATE_RESERVATION,
```

Comment:
Olisiko tässä oikeampaa käyttää `DELETE`?
```suggestion
                        Action.CalendarEventTime.DELETE,
```